Oracle/PCORI_MEDS_SCHEMA_CHANGE_ora.sql

```diff
@@ -40,11 +82,12 @@ with ndc(c_fullname,pcori_ndc,c_hlevel) as
     select m.c_fullname,ndc.pcori_ndc,m.c_hlevel from pcornet_med m
     inner join ndc on ndc.c_fullname=m.c_path where m.pcori_ndc is null
 ),
-ndcd as ( select c_fullname,pcori_ndc, row_number() over (partition by C_FULLNAME order by c_hlevel desc) row from ndc)
```

Comment:
I believe this change and the next one aren't actually changes? Can you remove these, it's hard to read.